src/vs/workbench/contrib/notebook/browser/contrib/editorStatusBar/editorStatusBar.ts

```diff
@@ -305,9 +305,9 @@ export class NotebookIndentationStatus extends Disposable implements IWorkbenchC
 		}
 
 		const cellEditorOverridesRaw = editor.notebookOptions.getDisplayOptions().editorOptionsCustomizations;
-		const indentSize = cellEditorOverridesRaw['editor.indentSize'] ?? cellOptions?.indentSize;
-		const insertSpaces = cellEditorOverridesRaw['editor.insertSpaces'] ?? cellOptions?.tabSize;
-		const tabSize = cellEditorOverridesRaw['editor.tabSize'] ?? cellOptions?.insertSpaces;
+		const indentSize = cellEditorOverridesRaw?.['editor.indentSize'] ?? cellOptions?.indentSize;
+		const insertSpaces = cellEditorOverridesRaw?.['editor.insertSpaces'] ?? cellOptions?.tabSize;
+		const tabSize = cellEditorOverridesRaw?.['editor.tabSize'] ?? cellOptions?.insertSpaces;
 
 		const width = typeof indentSize === 'number' ? indentSize : tabSize;
 
```

Comment:
> const message = insertSpaces ? `Spaces: ${width}` : `Tab Size: ${width}`;

This is in the next line.
Now that we have types, not sure `Tab Size true` makes sense,
I would recommend reviewing this message, hence created a new issue

Reply:
https://github.com/microsoft/vscode/issues/209496

Reply:
new commit should resolve this. looks like a typo initially that had the settings swapped. width should always be a number, decided on by indentSize either being a number or tabSize signaling to get the width from that setting.